### DISCUSSION TITLE
1920x1080 and 3840x2160 resolution please.

### DISCUSSION THREAD

**Discussion by AEAirEmpire** [lllyasviel/Fooocus#2005] [Ideas]
Please add 1920x1080/16:9 and 3840x2160/16:9 resolution of generated images to Focus.
Thank you.
<img width="364" alt="1" src="https://github.com/lllyasviel/Fooocus/assets/102669069/6c9c69e8-de1f-475c-bc91-27cdd6175243">

**Comment by xhoxye** [comment]
Please set it yourself in config.txt

**Reply by AEAirEmpire** [reply]
Everything worked out! Thank you. I had to change two configuration files: C:\Fooocus\config.txt and C:\Fooocus\modules\config.py
<img width="1181" alt="2" src="https://github.com/lllyasviel/Fooocus/assets/102669069/04547be4-554e-459f-81a5-534b9ebdafa6">
<img width="1202" alt="3" src="https://github.com/lllyasviel/Fooocus/assets/102669069/2345ac43-db92-4c86-ba50-a914a100587f">
<img width="396" alt="4" src="https://github.com/lllyasviel/Fooocus/assets/102669069/cb309f22-bf5f-4015-b223-8757e8ec4ae5">

**Reply by mashb1t** [reply]
this is not going to/will not work well as SDXL has not been trained on these resolutions (see https://github.com/lllyasviel/Fooocus/pull/1617), but all the best if it's sufficient for you.

Hint: better results can be achieved by generating an image and upscaling it afterwards x2 in Fooocus.

**Reply by AEAirEmpire** [reply]
Good day!
![2024-01-21_14-51-04_7167](https://github.com/lllyasviel/Fooocus/assets/102669069/ee032c5e-cf1d-470c-b455-929d8dccb0ad)
![2024-01-21_14-59-12_5467](https://github.com/lllyasviel/Fooocus/assets/102669069/4ac2051c-efbe-4e4e-a0a9-ce07cf1b7d46)
![2024-01-21_14-54-49_7602](https://github.com/lllyasviel/Fooocus/assets/102669069/6790a7d2-59f4-40d7-a899-d26469a42ded)
![2024-01-21_15-06-50_4049](https://github.com/lllyasviel/Fooocus/assets/102669069/606e444c-085c-442d-97e0-ffabc0de39c0)
![2024-01-21_15-11-48_8405](https://github.com/lllyasviel/Fooocus/assets/102669069/a5caa93e-fbbc-4f6a-8dec-7f11f6bb829c)
![2024-01-21_15-27-04_2033](https://github.com/lllyasviel/Fooocus/assets/102669069/0b6cb925-d6a8-481a-bc33-40a504098cf9)

<img width="380" alt="prs" src="https://github.com/lllyasviel/Fooocus/assets/102669069/42944f77-e644-4a9d-b4fa-ee626e954d91">

Everything works, very well. In resolution 1920x1080 (16x9)

Yes, I can generate an image and scale it twice. But this doesn’t suit me, because Foookus doesn’t have the original 16*9 resolution option, at least one.

Thanks for your help and your time.

**Reply by metercai** [reply]
Suggest placing often used aspect ratios at the front like this and prompt @mashb1t 
<img width="317" src="https://github.com/lllyasviel/Fooocus/assets/5652458/01cd1af4-749f-48ce-8145-874801d0099e">

**Reply by AEAirEmpire** [reply]
Thx. Yes, it will be very close. I didn't know that.
1152x3=3456
896x3=2688
As a result, we have 3456x2688. A little more than 4K. This is if we assume that 4K considers it to be 2160p

**Reply by mashb1t** [reply]
@metercai Prioritizing resolutions in the ordered list might lead to confusion for new users as the list isn't ordered anymore, and i'd suggest not to implement the sort from styles for resolutions.
Feel free to adjust the order to your liking using the config.

**Reply by metercai** [reply]
I see your concern, maybe you're right, but I think it's better that the often’s was placed at the front from various of things :)

**Reply by K3nt3L** [reply]
3840x2160 is a resolution that goes far beyond the capabilities of SDXL models. In addition, it requires a monstrous GPU. The maximum size generated by an SDXL model is 2048x1024 (with Tempest 0.1).
The proposed ratios are constrained by computation, not software.
Read this  :
https://www.reddit.com/r/StableDiffusion/comments/15c3rf6/sdxl_resolution_cheat_sheet/

**Comment by poor7** [comment]
For some things, 1920x1080 works more than well!
Examples:
![8398026627928060391](https://github.com/lllyasviel/Fooocus/assets/154618384/40b689fc-4b10-4a87-ae0d-1237f03a9d84)
![image (15)](https://github.com/lllyasviel/Fooocus/assets/154618384/0e577954-3f8a-48eb-a4a3-92db26a805fe)

**Reply by AEAirEmpire** [reply]
Wow, the second picture is very cool 👍  
The first picture is of course more realistic, but the sun just doesn’t look very realistic. But this is already a trifle that can be corrected with the correct prompt request.

**Reply by ghost** [reply]
1920x1080 and especially 1080x1920 didn't work well in Stable diffusion. Try generating a person, and you will understand why. Anyway, you can easily add it yourself. Just edit your config.txt file.

**Reply by poor7** [reply]
What kind of person should I create?
![image (36)](https://github.com/lllyasviel/Fooocus/assets/154618384/f067dc3a-b2bd-4ccf-9784-67a5f4928f71)
![image (33)](https://github.com/lllyasviel/Fooocus/assets/154618384/e445c7cc-dfb4-4061-b44d-d4a125257d2e)
![image (43)](https://github.com/lllyasviel/Fooocus/assets/154618384/9336f4db-df92-467c-bad0-88475c5dd3ae)

**Reply by ghost** [reply]
Didn't you notice that the body and even the face are vertically stretched?

**Comment by AEAirEmpire** [comment]
The most popular monitor aspect ratios include 4:3, 16:9 and 21:9.

It's strange, of course, that Fucus doesn't have a 16:9 standard. 4:3 and 21:9 are in Foookus.

**Reply by mashb1t** [reply]
See https://github.com/lllyasviel/Fooocus/discussions/117#discussioncomment-7364981, 16:9 is just not in the standard training dataset of SDXL.

**Comment by xhoxye** [comment]
![QQ截图20240122094504](https://github.com/lllyasviel/Fooocus/assets/129571231/23e37374-4028-4980-ae30-ed0713ef962e)

**Reply by elpolax** [reply]
Even though I update the "config.txt" file to have an additional aspect ratio, nothing changes. When you run run.bat they don't appear. What could be the problem?

**Reply by mashb1t** [reply]
@elpolax please check the terminal output for formatting errors in the config, most likely you have a comma after the last resolution item of the list.

**Reply by mschmidt-ffstudios** [reply]
Another option is also to add the values in the flag.py directly in the subfolder - /modules/flags.py

**Comment by leopardo** [comment]
You probably moved the folder around after the installation.

**Comment by K3nt3L** [comment]
If you want to generate images in a cinematic format, a model has been specifically trained on this ratio: **Chinook**. Moreover, its style is oriented towards cinematic panoramas. 
[Chinook download](https://civitai.com/models/400589/chinook)
Free Ad for the author  ;)
 "Chinook is designed to produce stunningly realistic, cinematic images that look like they've been plucked straight from the silver screen".
![Chinook_00520-1460561509](https://github.com/user-attachments/assets/19278b82-d6c4-410a-9bb5-fc66719a12af)
by  [Aurety](https://civitai.com/user/Aurety)
advice : 1536x640 and HiRes, and/or upscale
triggers : cinematic, movie still, filmgrab
